packages/camera/camera_android/CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 ## 0.10.5
 
+* Adds support for NV21 as a new streaming format in Android which includes correct handling of image padding when present.
+
```

Comment:
Nit: remove extra space

Reply:
Fixed in 76dd696960fb6f35467e0cc763afec425cde5de1

Reply:
Fixed in 76dd696960fb6f35467e0cc763afec425cde5de1